autoload/neomake/makers/ft/markdown.vim

```diff
@@ -19,6 +19,13 @@ function! neomake#makers#ft#markdown#proselint()
                 \ }
 endfunction
 
+function! neomake#makers#ft#markdown#writegood()
+    return {
+                \ 'args': ['--parse'],
+                \ 'errorformat': '%f:%l:%c:%m'
```

Comment:
Should specify the type, e.g. %W.
See also https://github.com/neomake/neomake/issues/584.